backend/src/main/java/eventlistener/service/event/EventMapper.java

```diff
@@ -4,29 +4,45 @@
 import eventlistener.model.event.EventToModifyDTO;
 import eventlistener.model.event.ResponseEventDTO;
 import eventlistener.model.notificationuser.NotificationUser;
-import eventlistener.service.notificaionuser.NotificationUserService;
 import org.springframework.stereotype.Service;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 public class EventMapper {
 
-    public List<ResponseEventDTO> mapDtoToEvent(List<Event> events){
+    public List<ResponseEventDTO> mapEvent(List<Event> events){
```

Comment:
```suggestion
    public static List<ResponseEventDTO> mapEvent(List<Event> events){
```
and all the other mapper methods as well as long as you don't need any non static context in them